### DISCUSSION TITLE
Saving the files locally using uploader

### DISCUSSION THREAD

**Discussion by neelashkannan** [google/mesop#436] [Q&A] [answered]
can anyone say how to save a file locally using the uploader

**Comment by wwwillchen** [comment, marked as answer]
For https://google.github.io/mesop/components/uploader/ - the UploadEvent has a `file` attribute and you can save that locally using the regular Python way, e.g.

```python
with open(file_path, 'wb') as file:
    file.write(event.file.getvalue())
```
